From 14e0debcf41595ee96c1fa62925b1b3059116acd Mon Sep 17 00:00:00 2001
From: Anca-Jeanina Floarea <anca.floarea@freescale.com>
Date: Wed, 14 Nov 2012 21:27:34 +0000
Subject: [PATCH 419/518] dpa_ipsec: Remove build warning - unused variable

The variable "ret" is not used in "free_sa_mng" and variable
definition should be placed in the beginning of the function.

Signed-off-by: Anca Jeanina FLOAREA <anca.floarea@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 drivers/staging/fsl_dpa_offload/dpa_ipsec.c      |    4 ++--
 drivers/staging/fsl_dpa_offload/dpa_ipsec_desc.c |    1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
index 52466f3..4519d84 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
@@ -1016,6 +1016,8 @@ static void free_sa_mng(struct dpa_ipsec *dpa_ipsec)
 	struct dpa_ipsec_sa_mng *sa_mng;
 	struct inpol_tbl *pol_tbl, *tmp;
 	struct list_head *head;
+	struct list_head *pos, *n;
+	struct ipsec_manip_node *node;
 	int i = 0;
 
 	/* sanity checks */
@@ -1085,8 +1087,6 @@ static void free_sa_mng(struct dpa_ipsec *dpa_ipsec)
 	}
 
 	/* cleanup hmanips */
-	struct list_head *pos, *n;
-	struct ipsec_manip_node *node;
 	list_for_each_safe(pos, n, &dpa_ipsec->sa_mng.ipsec_manip_node_list) {
 		node = container_of(pos, struct ipsec_manip_node, ipsec_manip_node_list);
 		list_del(&node->ipsec_manip_node_list);
diff --git a/drivers/staging/fsl_dpa_offload/dpa_ipsec_desc.c b/drivers/staging/fsl_dpa_offload/dpa_ipsec_desc.c
index 7a583e0..ddb719c 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_ipsec_desc.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_ipsec_desc.c
@@ -1131,7 +1131,6 @@ int build_extended_decap_shared_descriptor(struct dpa_ipsec_sa *sa,
 		 esp_length, iv_length, icv_length, max_pad, data;
 	dma_addr_t dma_extra_cmds;
 	struct device *jrdev;
-	int ret;
 
 	desc = (uint32_t *)sa->sec_desc->desc;
 
-- 
1.7.5.4

